OpenSim/Tests/Future/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# This folder holds example code that shows what we *want* to be able to do
+# with OpenSim in the future. There is an executable target for every
+# C++ file whose name begins with `future`, but these executables are not
+# built by default (not part of the "all" target).
+
+# These executables are *not* tests.
```

Comment:
…then why do they live in the Tests directory? What about OpenSim/Future, or perhaps OpenSim/Examples/Future if you think most of the pseudocode will eventually become examples.